src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -776,4 +783,25 @@ public static int getPoLineLimit(JsonObject config) {
   public static CompositePurchaseOrder convertToCompositePurchaseOrder(JsonObject poJson) {
     return poJson.mapTo(CompositePurchaseOrder.class);
   }
+
+  public static boolean inventoryUpdateNotRequired(CompositePoLine compPOL) {
+    return (compPOL.getPhysical() == null && compPOL.getEresource() == null) ||
+      (compPOL.getPhysical() != null && compPOL.getPhysical().getCreateInventory() == Physical.CreateInventory.NONE && compPOL.getEresource() == null) ||
+      (compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == Eresource.CreateInventory.NONE && compPOL.getPhysical() == null) ||
+      (compPOL.getPhysical() != null && compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == Eresource.CreateInventory.NONE && compPOL.getPhysical().getCreateInventory() == Physical.CreateInventory.NONE) ||
+      // also ignore inventory interaction in case of "Other" order format
+      compPOL.getOrderFormat() == CompositePoLine.OrderFormat.OTHER;
+  }
+
+  public static boolean isHoldingsUpdateRequired(CompositePoLine compPOL) {
+    return (compPOL.getEresource() != null &&
```

Comment:
This conditional is quite large... We might consider breaking it up, or at least adding additional comments

Reply:
updated conditional code for better readability